dist/index.d.ts

```diff
@@ -54,8 +56,24 @@ interface IPosisProcessRegistry {
 	// if your bundle consists of several programs you can pretend that we have a VFS: "ANI/MyBundle/BundledProgram1"
 	register(imageName: string, constructor: new (context: IPosisProcessContext) => IPosisProcess): boolean;
 }
+interface IPosisCooperativeSchedulingV1 {
```

Comment:
Unimplemented yield is worse than no yield. Process can't detect when it needs to fall back to manual yield by doing `if (!budgetLeft) return`

Ah, you can make optional functions in ts. I wanted to avoid this symtax `yield?: (cb: () => void) => void`. :). I'll change it.